### PR TITLE
Hidden gem class

### PR DESCRIPTION
**Pull request by sehhajdkg**
Small PR, just adding a Hidden Gem constructor that takes in the Place and PlaceDetails are objects. Looks cleaner I think :)

### CODE REVIEW

**Review by helenatran — APPROVED** [review]

File: src/main/java/com/google/sps/data/Places.java [hunk 1]
```diff
@@ -198,22 +198,7 @@
...
-          String.valueOf(placeDetails.priceLevel), 
-          place.rating, 
-          place.userRatingsTotal, 
-          placeDetails.website, 
-          place.openingHours, 
-          place.photos[0].photoReference, 
-          place.photos[0].htmlAttributions, 
-          place.permanentlyClosed, 
-          place.businessStatus));
+        hiddenGems.add(new HiddenGem(place, placeDetails));
```
Comment (jamila-sabaz) [inline src/main/java/com/google/sps/data/Places.java:201]: this might be a silly question but why did you remove all the place details?

Comment (sehhajdkg) [reply src/main/java/com/google/sps/data/Places.java:201]: not a silly question at all. I haven't removed the place details, I send the enter place details object as a param and then set the relevant values in the hidden gem object. like hidden.priceLevel = placeDetails.priceLevel.

Comment (jamila-sabaz) [reply src/main/java/com/google/sps/data/Places.java:201]: makes sense ! thanks for explaining

**Review by lorikeet-s — APPROVED** [review]
Perfect, exactly what I was talking about, it moves all the complexity into the class that really understands hidden gems and doesn't make the caller worry about it.
